.circleci/config.yml

```diff
@@ -15,11 +15,11 @@ workflows:
           tag: latest
           after_build:
             - run:
-                name: Tag and Push with $CIRCLE_BUILD_NUM
+                name: Tag and Push with $CIRCLE_SHA1
```

Comment:
Yeah, Ive always thought it was weird that we used the build number instead of the git SHA. I'm good with this change 👍